.github/workflows/ci-push.yml

```diff
@@ -121,9 +127,11 @@ jobs:
       working-directory: ./racket/src
       run: make -j  $((cpus+1)) install
     - name: Tarballing
+      if: matrix.cc == 'gcc' || matrix.os == 'macos-latest'
```

Comment:
Why do we explicitly need to test this here?

Reply:
We do not need to test racket build with `gcc` and `clang`. We build with `clang` but do not test the build. We test the `gcc` one. So we only tarball the `gcc` version. At the same time, `macos` only builds with `clang`, so we tarball in that case as well.

Happy to change this if you think it's worth it.

Reply:
That seems fine (the odds of finding a miscompilation in clang that causes test failures is fine). Can you add a comment to that line about why?